addons/hr/wizard/hr_plan_wizard.py

```diff
@@ -1,49 +1,106 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from odoo import fields, models, _
+from odoo import api, fields, models, _
+from odoo.exceptions import ValidationError
 
 
 class HrPlanWizard(models.TransientModel):
     _name = 'hr.plan.wizard'
     _description = 'Plan Wizard'
 
     def _default_plan_id(self):
-        employee = self.env['hr.employee'].browse(self.env.context.get('active_id'))
+        # We know that all employees belong to the same company
+        employee = self.env['hr.employee'].browse(self.env.context.get('active_ids')[0])
```

Comment:
To prevent any issues if - somehow - the wizard is called without `active_ids`, what do you think? :innocent: 

```suggestion
        employee = self.env['hr.employee'].browse(self.env.context.get('active_ids', [])[:-1])
```